### ISSUE TITLE
Error when format is different than `html`

### ISSUE THREAD

**Issue by jamming** [Kotlin/dokka#67]
I'm using Dokka 0.9.7. With this configuration:

```
dokka {
    outputFormat = 'markdown' // with 'jekyll' too
    outputDirectory = "$buildDir/$projectSite/dokka"
    processConfigurations = [ 'compile' ]
}
```

I'm getting the following error:

```
Execution failed for task ':dokka'.
> Unable to create injector, see the following errors:
  
  1) org.jetbrains.dokka.JekyllFormatService has more than one constructor annotated with @Inject. Classes must have either one (and only one) constructor annotated with @Inject or a zero-argument constructor that is not private.
    at org.jetbrains.dokka.JekyllFormatService.class(JekyllFormatService.kt:6)
    at org.jetbrains.dokka.Utilities.DokkaModule.configure(DokkaModule.kt:43)
```

**Comment by yole** [comment]
Duplicate of #49; fixed already, fix will be included in 0.9.8

**Comment by jamming** [comment]
Thanks!
